src/commands/edge_app_server.rs

```diff
@@ -227,16 +285,21 @@ settings:
         assert!(content.contains("metadata: {")); // Make sure metadata is in its own section
         assert!(content.contains("\"srly-t6kb0ta1jrd9o0w\"")); // Checking an example metadata value
         assert!(content.contains("key"));
+        assert!(content.contains("[\"37.3861\", \"-122.0839\"]"));  // Make sure arrays are written correctly
```

Comment:
Let's test the whole file content, please. 

instead of contains using strict eq. 

I did this approach and it was an error since it turned out to cause bugs let's not use it further.

Reply:
OK, let's do this one as a follow-up. I've done it but it kept invalidating the valid string generated by itself. Will take a deeper look after that. Approve and release as it is.

Reply:
I can't approve when the test for the actual content does not pass :( Maybe someone else will

Reply:
I agree with @sergey-borovkov, just do one equality test to a full file content.

Reply:
OK, how a test could fail even with its own output? Would you like me to wait to figure that bug out? I'm copying its left output to the right, and then assert it.

Reply:
Done.